lib/ssf/ssf_methods.rb

```diff
@@ -18,6 +20,7 @@ def finish(time: nil)
       packet = Ssf::SSFSpan.encode(self)
 
       @client.send_to_socket(packet)
+      # @socket.send(packet, 0)
```

Comment:
Sort of - I remember you mentioned we wanted to remove the dependence on having a `client` from the `span`, but I wasn't 100% clear on what we wanted so I added support but then commented out most of the changes.